packages/apps/docs/package.json

```diff
@@ -57,7 +57,7 @@
     "mdast-util-to-markdown": "~1.5.0",
     "mdast-util-to-string": "~3.2.0",
     "mobx": "~6.9.0",
-    "next": "13.5.3",
+    "next": "13.4.5",
```

Comment:
what is the reason for downgrading next version here? 

cc @sstraatemans

Reply:
We have a policy in the repo to make sure the version of a package is the same across the monorepo. First I tried to upgrade the others, but got some error, so then downgraded this one.

Reply:
hm the new version was helping us with some performance.
not really happy with this one

Reply:
I think it makes sense for packages like `next` to have an exception, this is something close to the app itself, you don't want to force the whole repo to upgrade it at once every time we upgrade in the future. Already proven by the fact that you had troubles doing it now.

`next` isn't some bundled dependency that is going to give issues in builds or increase bundle sizes.

Reply:
Next.js has a dependency on React. Component libs too. So to prevent any issues here (we've had them before) we should stay in sync, _especially_ libs like Next.js + React.